---
bool is_lift_open